Templates/FaceLocator-NodeJS/metadata.json

```diff
@@ -11,5 +11,6 @@
         "path",
         "connection",
         "tableName"
-    ]
+    ],
+    "disabledInTryMode": false
```

Comment:
Wondering if we shouldn't invert the flag to be "tryModeEnabled" and only explicitly set that flag to true for the ones we support in try mode. That way it's opt in, and all templates are disabled in try mode unless we explicitly set this. That makes it less likely that someone accidentally adds a template that shows up but doesn't work in try mode.